app/views/description/walks/show.html.erb

```diff
@@ -42,8 +42,8 @@
               <span>(<%= @walk.reviews.size%>)</span>
               <% end %>
             </div>
-          <p><strong><%= @walk.length/1000 %></strong> km</p>
-          <p><strong><%= @walk.duration/60 %></strong> min</p>
+          <p><i class="fa-solid fa-person-hiking"></i> <strong><%= @walk.length/1000 %></strong> km</p>
```

Comment:
Oh, cool, il faudrait que je l'ajoute aussi sur la page de recap and review du coup. :-)

Reply:
carrément très bonne idée !